text/0003-bundle-change.md

```diff
@@ -0,0 +1,107 @@
+- Feature Name: bundle\_change
+- Start Date: 2018-07-22
+- RFC PR: (leave this empty)
+- Bundler Issue: (leave this empty)
+
+# Summary
+
+This RFC proposes a new bundle change command. The idea behind it is to make Gemfile editable via command line.
+
+# Guide-level explanation
+
+At an abstract level the change command is a composite of remove and add command. the requested gem would be first removed and then added with the supplied options (which it supports).
```

Comment:
> the requested gem would be first removed and then added with the supplied options

This needs more explanation on what this does and why removing and then adding the gem is better than replacing it.

Reply:
It was the initial idea to remove the line which depends on `remove` and then add it.